src/network/stk_host.cpp

```diff
@@ -375,6 +375,7 @@ uint8_t* STKHost::receiveRawPacket(const TransportAddress& sender,
             TransportAddress a(m_host->address);
             Log::verbose("STKHost", "No answer from the server on %s",
                          a.toString().c_str());
+            delete[] buffer;
```

Comment:
I've applied this to the networking branch (6946306c4c29f23998faf4b2807cc22b629fb057). Thanks